pilot/pkg/bootstrap/istio_ca.go

```diff
@@ -363,18 +336,15 @@ func (s *Server) createIstioCA(client corev1.CoreV1Interface, opts *caOptions) (
 	if _, err := os.Stat(signingKeyFile); err != nil && client != nil {
 		// The user-provided certs are missing - create a self-signed cert.
 		log.Info("Use self-signed certificate as the CA certificate")
-		// Abort after 20 minutes.
```

Comment:
Why is the following code removed?
		// Abort after 20 minutes.
		ctx, cancel := context.WithTimeout(context.Background(), time.Minute*20)
		defer cancel()

Reply:
In `NewSelfSignedIstioCAOptions`, the logic is:

1. get CA secret
2. if not found, sign root cert and create CA secret.
3. If CA secret create fails, then istiod will error exit. 

This flow is like most modules we have currently.

Reply:
Thanks! Can you share a design document describing the changes the PR bring to the CA flows and present it in a Security Work Group meeting (https://docs.google.com/document/d/12Xz8fCuql2tPL-cpdLXGGE44kYT4o9NB6mljSiS_01U/edit#heading=h.o8pz6aqnzzgk)?

Reply:
I donot think i can attend the meeting as the time is too late for my time zone.